generic/tests/ioquit.py

```diff
@@ -26,16 +30,32 @@ def run(test, params, env):
     session2 = vm.wait_for_login(timeout=login_timeout)
 
     bg_cmd = params.get("background_cmd")
-    error.context("Add IO workload for guest OS.", logging.info)
+    error_context.context("Add IO workload for guest OS.", logging.info)
     session.cmd_output(bg_cmd, timeout=60)
 
-    error.context("Verify the background process is running")
+    error_context.context("Verify the background process is running")
     check_cmd = params.get("check_cmd")
     session2.cmd(check_cmd, timeout=360)
 
-    error.context("Sleep for a random time", logging.info)
+    error_context.context("Sleep for a random time", logging.info)
     time.sleep(random.randrange(30, 100))
     session2.cmd(check_cmd, timeout=360)
 
-    error.context("Kill the VM", logging.info)
+    error_context.context("Kill the VM", logging.info)
     vm.process.close()
+
+    error_context.context("Check img after kill VM", logging.info)
+    base_dir = data_dir.get_data_dir()
+    image_name = params.get("image_name")
+    image = qemu_storage.QemuImg(params, base_dir, image_name)
+    try:
+        image.check_image(params, base_dir)
+    except exceptions.TestWarn as e:
+        if "Leaked clusters" in e.message:
+            error_context.context("Fix cluster leak issue", logging.info)
+            restore_cmd = params.get("image_restore_cmd") % image.image_filename
+            cmd_status = process.system(restore_cmd)
+            if cmd_status:
+                test.fail("Failed to restore cluster leak of image")
```

Comment:
It will be better, if the message is changed to "Failed to repair cluster leaks on the image"